src/dragonball/src/device_manager/legacy.rs

```diff
@@ -42,6 +44,10 @@ pub enum Error {
 pub struct LegacyDeviceManager {
     #[cfg(target_arch = "x86_64")]
     i8042_reset_eventfd: EventFd,
+    #[cfg(target_arch = "aarch64")]
+    pub(crate) _rtc_device: Arc<Mutex<RTCDevice>>,
```

Comment:
Does it have any special meaning that adding an underscore as a prefix of the `_rtc_device`?

Reply:
Thanks for your question! @liubin In rust, if a variable is not in use, it should add an underscore as a prefix. We use `_rtc_device` here as a placeholder for extending it in future.